### ISSUE TITLE
Add a11y attributes to existing components.

### ISSUE THREAD

**Issue by stepan-anokhin** [benetech/VideoDeduplication#102]
Currently most of the components are missing accessibility attributes. This should be fixed until there are too many of them.
